### PR TITLE
chore(docs): update CONTRIBUTING

### PR DESCRIPTION
**Pull request by hacker65536**
Remove description about `install.sh`

related #4588

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -364,7 +363,7 @@ $ docker run -v $(pwd):/app -w /app aws-cdk <CDK ARGS>
 In many cases, you don't really need to build the entire project. Say you want to work on the `@aws-cdk/aws-ec2` module:
 
 ```console
-$ ./install.sh
+$ ./build.sh
```
Comment (nmussy) [inline CONTRIBUTING.md]: It's not really approriate to replace `install.sh` with `build.sh` here. The goal of this section is to only build a specific package and its dependencies.

Looks like a simple `yarn install` would do the trick

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -468,7 +467,7 @@ Cycle: @aws-cdk/aws-sns => @aws-cdk/aws-lambda => @aws-cdk/aws-codecommit => @aw
 To update all dependencies (without bumping major versions):
 
 1. Obtain a fresh clone from "master".
-2. Run `./install.sh`
+2. Run `./build.sh`
```
Comment (nmussy) [inline CONTRIBUTING.md]: Same comment here, `yarn install` might be sufficient?

File: CONTRIBUTING.md (hunk 1 above)
Comment (hacker65536) [reply CONTRIBUTING.md]: @nmussy  Thank you for your advice. I'll replace it.

**Comment by nmussy** [comment]
Tagging @RomainMuller to correct me if I'm wrong

**Comment by nmussy** [comment]
There is also a mention of the removed `package-lock.json` in CONTRIBUTING:

> Make sure package-lock.json files are included in your commit.

Seen as there's only a single `yarn.lock` at the root of the monorepo, I'm not sure how to change that one

**Pull request merged**